scene/resources/animation.cpp

```diff
@@ -870,6 +882,7 @@ void Animation::_get_property_list(List<PropertyInfo> *p_list) const {
 		p_list->push_back(PropertyInfo(Variant::DICTIONARY, "_compression", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_NO_EDITOR | PROPERTY_USAGE_INTERNAL));
 	}
 	p_list->push_back(PropertyInfo(Variant::ARRAY, "markers", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_NO_EDITOR | PROPERTY_USAGE_INTERNAL));
+	p_list->push_back(PropertyInfo(Variant::ARRAY, "collapsed_groups", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_NO_EDITOR | PROPERTY_USAGE_INTERNAL));
```

Comment:
This seems unfortunate to store in the actual serialized `Animation` resource, since it would then have an actual runtime impact as well. Just removing this and the corresponding `_set` and `_get` code seems to function just as well, with the caveat that the collapsed state isn't preserved when closing the scene/editor, which seems fine.

I guess this could be a viable candidate for `EditorSettings::set_project_metadata`, but I'm not seeing any resource-specific data being stored in there, so maybe not?